tests/OAuth2Test.php

```diff
@@ -0,0 +1,29 @@
+<?php
+namespace FuelSdk\Test;
+use FuelSdk\ET_Client;
+use FuelSdk\ET_Asset;
+use PHPUnit\Framework\TestCase;
+
+final class OAuth2Test extends TestCase
+{
+    private $client;
+
+    function __construct()
+    {
+        $this->client = new ET_Client(true);
+    }
+
+    public function testIfAuthTokenAndRefreshTokenDifferIfRefreshTokenIsEnforced()
+    {
+        $token = $this->client->getAuthToken();
+        $refreshToken = $this->client->getRefreshToken(null);
+        $this->client->refreshTokenWithOAuth2(true);
+
+        $newtoken = $this->client->getAuthToken();
+        $newrefreshToken = $this->client->getRefreshToken(null);
+
+        $this->assertNotEquals($token, $newtoken);
```

Comment:
can we split this test in 2? The best practice would be to have 1 assert per test

Reply:
I would say that this is OK since the asserts are made for the same concern - state change (authToken and refreshToken) caused by the get refreshTokenWithOAuth2(true) method invocation. In another test that invokes the method with false, only authToken is affected, hence we can have only one assert for this case.

Reply:
then maybe we can compare 2 arrays composed of accessToken & refreshToken so we have 1 assert